src/main/java/yevhen/bookstore/security/JwtUtil.java

```diff
@@ -0,0 +1,60 @@
+package yevhen.bookstore.security;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jws;
+import io.jsonwebtoken.JwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import java.nio.charset.StandardCharsets;
+import java.util.Date;
+import java.util.function.Function;
+import javax.crypto.SecretKey;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JwtUtil {
+    private SecretKey secret;
+
+    @Value("1500000")
```

Comment:
it should be in application properties file and use reference here

Reply:
I don't know why, but with such references ${jwt.expiration} and ${jwt.secret} mvn clean package can't be succesfully builded. It throws such exception -> Could not resolve placeholder 'jwt.secret' in value "${jwt.secret}"
Despite this application can start and work. But i have to keep previous version with hard-coding of this values to match the git cheking

Reply:
Ok. It's strange, I think you should figure out what the cause of the issue

Reply:
@yevhenkukharchuk the reason may be that you forgot to declare these properties in test profile (`application.properties` under `test` folder) 

try checking it